status/hostcheck.go

```diff
@@ -143,6 +145,30 @@ func (c *writableCheck) Run() error {
 	return nil
 }
 
+type forceErrorCheck struct {
+	path string
+}
+
+func (c *forceErrorCheck) Kind() string {
+	return "forceError"
+}
+
+func (c *forceErrorCheck) Name() string {
+	return fmt.Sprintf("forceError-%s", c.path)
+}
+
+func (c *forceErrorCheck) Run() error {
+	fileName := strings.Join([]string{
```

Comment:
You can use `filepath.Join` from the `path/filepath` pkg here.

Reply:
Done! :)